.eslintrc.yml

```diff
@@ -33,6 +33,10 @@ rules:
   import/no-extraneous-dependencies: error
   prettier/prettier: error
   no-console: off
+  import/no-unresolved:
+    - error
+    - ignore:
+        - '@mdx-js/react'
```

Comment:
why are eslint rules, updating in a documentation change?

Reply:
Because I found it’s not working actually.